### PR TITLE
Support to split many records

### PR DESCRIPTION
**Pull request by koxudaxi**
The PR supports split many records over 1000 lines when using `batch_execute`.

### CODE REVIEW

File: pydataapi/pydataapi.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 )
 
 import boto3
+from more_itertools import chunked, flatten
```
Comment (Rubyj) [inline pydataapi/pydataapi.py]: Adding this package has made my chalice lambda go above 50MB 😆 I thought it would be funny to share 🤣

Comment (koxudaxi) [reply pydataapi/pydataapi.py]: Really?
The package looks small which is 57.3 kB 🤔 
https://pypi.org/project/more-itertools/#files

Comment (Rubyj) [reply pydataapi/pydataapi.py]: > Really?
> The package looks small which is 57.3 kB 🤔
> https://pypi.org/project/more-itertools/#files

Yeah. I think my lambda was very very very close to 50MB. I will have to clean some things out if I can.

Comment (koxudaxi) [reply pydataapi/pydataapi.py]: OK, I will create a new issue to reduce the size of dependency packages.

**Pull request merged**